starknet_py/tests/e2e/contract_interaction/interaction_test.py

```diff
@@ -219,7 +219,7 @@ async def test_calculate_hash_without_max_fee():
     contract = await Contract.from_address(contract.address, client)
     prepared_call = contract.functions["put"].prepare(key, value)
 
-    prepared_call.hash
+    prepared_call.hash  # pylint: disable=pointless-statement
```

Comment:
Why do we need this statement?

Reply:
If is throws when hash is not defined I suggest using `assert hasattr(prepared_call, "hash")` to be explicit.